lib/loadFiles.py

```diff
@@ -28,6 +28,7 @@ def write_caselist(logDir, List=None, Dir=None):
 
     if Dir is not None:
         imgs= glob(pjoin(Dir, '*.nii.gz'))
+        imgs.sort()
 
```

Comment:
The bug was discovered when testing `docker` and `singularity` images on WIN and MAC machines.